spec/kong_tests.conf

```
@@ -15,6 +15,9 @@ pg_host = 127.0.0.1
 pg_port = 5432
 pg_timeout = 10000
 pg_database = kong_tests
+# note: this does not trigger readonly mode to be enabled on its own
+# for that pg_ro_host is also needed
+pg_ro_user = kong_ro
```

minor thing:

```suggestion
pg_ro_user = kong_ro_tests
```

just to keep the pattern of having the `tests` entries in the DB separate from the regular ones, makes it easier to avoid mixing up manual and automated tests

---

Hmm, it seems that the current pattern for running integration tests is to run it with the `kong` user, therefore I used the `kong_ro` user for the readonly mode. Otherwise we will have to change `pg_user = kong_tests` as well to keep them consistent.

---

Ah, ok, we use the same user but a different database (it's still `kong` user but `kong_tests` for the database/keyspace).